test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -376,7 +376,7 @@ private void CategoricalFeaturizationOn(params string[] dataPath)
             );
 
             // Read the data.
-            var data = reader.Read(dataPath);
+            var data = reader.Read(new MultiFileSource(dataPath));
 
```

Comment:
If you change this file you also need to change the CookBook. [https://github.com/dotnet/machinelearning/blob/7faa46c41fcd1a214fac7bfbdc8a9d1adbcc8d41/docs/code/MlNetCookBook.md#mlnet-cookbook](https://github.com/dotnet/machinelearning/blob/7faa46c41fcd1a214fac7bfbdc8a9d1adbcc8d41/docs/code/MlNetCookBook.md#mlnet-cookbook)